### DISCUSSION TITLE
Support for downtime ?

### DISCUSSION THREAD

**Discussion by sebguilbaud** [TwiN/gatus#301] [Q&A] [answered]
sometimes services have a "good" reason to be down, scheduled downtime could be used to show services as "down" but a "softer down"

**Comment by sebguilbaud** [comment]
(I know it's a very nagios-ish way of thinking :-) )

**Comment by TwiN** [comment, marked as answer]
That sounds like a great idea!
There's already a `maintenance` configuration that lets you set a downtime, _but_ that feature is not reflected at all on the UI.

**Comment by sebguilbaud** [comment]
I missed the "maintenance" part of the docs :)

**Comment by CoreKit-glitch** [comment]
Hey,

Could this maintenance be reflected on the UI on a per asset style ? (specific color or state)

**Reply by PythonGermany** [reply]
I worked on a feature last month that adds the ability to reflect maintenance state in the UI: https://github.com/TwiN/gatus/pull/1445#issuecomment-3658182026
